test/system/030-run.bats

```diff
@@ -786,6 +786,17 @@ EOF
     user=$(id -un)
     run_podman 1 run --rm $IMAGE grep $user /etc/passwd
     run_podman run --hostuser=$user --rm $IMAGE grep $user /etc/passwd
+
+    # find a user with a uid > 100 that is a valid octal
+    # Issue #19800
+    octal_user=$(awk -F\: '$1!="nobody" && $3>100 && $3~/^[0-7]+$/ {print $1 " " $3; exit}' /etc/passwd)
```

Comment:
Can we just add/commit a user octal_user to a container image, are we guaranteed to have an image with such a user?

Reply:
wouldn't be able to run this test only with that image and cause it to fail on any other system where the user is not present?

Reply:
I am saying you do it with a Containerfile
```
from $IMAGE
run useradd -u 8000 test-user
```
podman build -t testimage .
podman run --user test-user testimage

Reply:
Ok so the host user account must exist for this to work. So only a container within a container would work.